lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_tools_path_helper.rb

```diff
@@ -0,0 +1,59 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  module Helper
+    module Android
+      # Helper to find the paths of common Android build and SDK tools on the current machine
+      # Based on `$ANDROID_SDK_ROOT` and the common relative paths those tools are installed in.
+      #
+      class ToolsPathHelper
+        attr_reader :android_sdk_root
+
+        def initialize(sdk_root: nil)
+          @android_sdk_root = sdk_root || ENV['ANDROID_HOME'] || ENV['ANDROID_SDK_ROOT'] || ENV['ANDROID_SDK']
```

Comment:
Good question, I don't remember where I found that from tbh, but I think I [borrowed that from `fastlane` itself](https://github.com/fastlane/fastlane/search?q=ANDROID_SDK)

> Also, the docs say that ANDROID_SDK_ROOT is deprecated.

Oh, funny, I always thought `ANDROID_HOME` was the one being deprecated and `ANDROID_SDK_ROOT` was the new hotness in town.

Given what you linked in the documentation but also the order those env vars are checked by `fastlane` (and thus by this line here which I borrowed from there), I guess I'll have to rename all my env vars in my machine, `~/.zshrc`, and personal scripts I have around on my Mac (and maybe in other scripts in this repo like [this one](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/tools/rename_apk_aab.sh#L33), and check which ones we use on our CI for our Android repos too)

> Should we avoid looking for it? 

We still definitively need to keep looking for it, because I think me and many Android devs still use `ANDROID_SDK_ROOT` in many places in their config on their machine.

Reply:
Fixed in 7cba9a00eb16cfdba8193e5e664f313b79e2dd8e

Then applied a similar fix in 97d32333ad679942d9d8d594139cd8ed04da8cee on other actions that predates this PR (`android_send_app_size_metrics`) but had the same incorrect assumption (that `ANDROID_HOME` was the deprecated one, while `ANDROID_SDK_ROOT` is the deprecated one and `ANDROID_HOME` is the newest)